File: PythonPackage/cmsis_stream/cg/scheduler/templates/cmsis.cpp

                  {

{% for ptr in ptrs %}
                   {{ptr[0]}}* {{ptr[1]}};
{% endfor %}
{% for ptr in inputs %}
                   {{ptr[0]}}={{ptr[1].access}}getReadBuffer({{ptr[2]}});
{% endfor %}
{% for ptr in outputs %}
                   {{ptr[0]}}={{ptr[1].access}}getWriteBuffer({{ptr[2]}});
{% endfor %}
                   {{func}}({{args}});
                   cgStaticError = 0;
                  }

File: PythonPackage/cmsis_stream/cg/scheduler/templates/cmsis.py

{% for ptr in inputs %}
       {{ptr[0]}}={{ptr[1].access}}getReadBuffer({{ptr[2]}})
{% endfor %}
{% for ptr in outputs %}
       {{ptr[0]}}={{ptr[1].access}}getWriteBuffer({{ptr[2]}})
{% endfor %}
       {{outArgs}}[:]=dsp.{{func}}({{inArgs}})
       cgStaticError = 0
       